### DISCUSSION TITLE
tutorials/ui/gui_using_fonts

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#278] [User-contributed notes]
# tutorials/ui/gui_using_fonts

Godot allows you to set specific fonts for different UI nodes. There are three different places where you can setup font usage. The first is the theme editor. Choose the node you want to set the fo...

https://docs.godotengine.org/en/4.3/tutorials/ui/gui_using_fonts.html

**Comment by PutridNonsense** [comment]
Feels like there should be a section on kerning pairs for bitmap fonts since it seems undocumented

**Reply by Calinou** [reply]
Please open an issue on https://github.com/godotengine/godot-docs/issues to request new documentation.

**Comment by MadeScientist** [comment]
Note: If you are making a pixel game but want to use MSDF font in it, you need to select `canvas_items` instead of `viewport` in stretch mode.

**Comment by admin** [comment]
*Originally posted by [**Nodragem**](https://github.com/Nodragem) on 2026-06-04 19:00:28 UTC*

> >  If you wish to use the default project font while displaying emoji, leave the Base Font property in FontVariation empty while adding a font fallback pointing to the emoji font
>
> This does not seem to work, the font fallback is used instead of the Open Sans SemiBold
